src/helpers/logger.ts

```diff
@@ -41,3 +41,30 @@ export function logError(message: string): void {
 export function info(message: string): void {
   console.log(`[${_getTimestamp()}] ['info'] ${message}`)
 }
+
+export class UploadLogger {
```

Comment:
A future PR may want to utilize [the built-in Console class](https://nodejs.org/api/console.html#class-console).